src/mixins/wallet.js

```diff
@@ -93,5 +106,48 @@ export default {
         'menubar=no,location=no,width=576,height=768'
       );
     },
+    async fetchFollowers() {
+      try {
+        await this.walletFetchFollowers();
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
+    },
+    exportFollowerList() {
```

Comment:
```suggestion
    async exportFollowerList() {
```

Reply:
This function doesn't use await and only returns a Promise, so do we still need to be declared as async？

Reply:
How about just use sync mode
https://csv.js.org/stringify/api/sync/